### PR TITLE
Results page

### PR DESCRIPTION
**Pull request by abhishektanwar**

### CODE REVIEW

File: CSS/quiz.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	align-items: center;
+}
+
+.question-container{
+	border-radius: 15px;
+	background-color: var(--dark-primary-color);
+	height: fit-content;
+	width: 60rem;
+	margin-top: 3rem;
+	padding: 30px;
```
Comment (itsmebipul00) [inline CSS/quiz.css]: Use rems ??

File: CSS/rules.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	align-items: center;
+	justify-content: center;
+}
+
+.rules{
+	background-color: var(--dark-primary-color);
+	height: 600px;
+	width: 700px;
+	border-radius: 15px;
+	padding: 20px;
```
Comment (itsmebipul00) [inline CSS/rules.css]: Use rems ??

File: Pages/answerPage.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+			</div>
+
+			<!-- LOGO -->
+			<div class="nav-logo nav-section-items">
+				<a class="link-no-style" href="./">
+					<div class="primary-text-color logo-title">QUIZO</div>
+				</a>
+			</div>
+			<ul class="nav-bar-links list-style-none nav-section-items">
+				<li class="list-inline-item avatar-in-nav-links">
```
Comment (itsmebipul00) [inline Pages/answerPage.html:36]: you could have used a shorter classname here

**Review by itsmebipul00 — APPROVED** [review]
Please try to give a smaller file for review next time! Unless it is renaming of files or rearranging of code

File: CSS/quiz.css (hunk 1 above)
Comment (abhishektanwar) [reply CSS/quiz.css]: fixed

File: CSS/rules.css (hunk 2 above)
Comment (abhishektanwar) [reply CSS/rules.css]: fixed.

File: Pages/answerPage.html (hunk 3 above)
Comment (abhishektanwar) [reply Pages/answerPage.html:36]: thank you for mentioning it.

File: CSS/quiz.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
+body{
+	background-color: var(--light-primary-color);
+}
+
+.quiz-container{
+	align-items: center;
+}
+
+.question-container{
+	border-radius: 15px;
```
Comment (dev-ashishsethi) [inline CSS/quiz.css:10]: you could have used rem

File: CSS/quizes.css [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+#next1,#next2{
+	position: absolute;
+	right: 3rem;
+	top: 0;
+	height: 100%;
+    opacity: 0.7;
+    background: var(--white-color);
+    color: var(--black-color);
```
Comment (dev-ashishsethi) [inline CSS/quizes.css:46]: make variable capital

File: CSS/quizes.css [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+	color: var(--white-color);
+	border: none;
+	outline: none;
+	padding: 10px;
+	cursor: pointer;
+}
+
+#container #posts{
+	max-width: 30rem;
+	min-height: 5rem;
```
Comment (dev-ashishsethi) [inline CSS/quizes.css]: don't do styling in ids

**Review by dev-ashishsethi — APPROVED** [review]
looks good but do check comments

File: CSS/quizes.css (hunk 6 above)
Comment (abhishektanwar) [reply CSS/quizes.css]: fixed

File: CSS/quiz.css (hunk 4 above)
Comment (abhishektanwar) [reply CSS/quiz.css:10]: I want to keep radius of 15px only so using absolute unit.

**Pull request merged**